File: app/src/main/java/com/example/floating/bakingapp/recipes/RecipeComponent.java
package com.example.floating.bakingapp.recipes;

import dagger.Subcomponent;

/**
 * Created by apple on 14/01/2018.
 */
@RecipeScope
@Subcomponent(modules = RecipeModule.class)
public interface RecipeComponent {
    void inject(RecipeActivity recipeActivity);
}

File: app/src/main/java/com/example/floating/bakingapp/recipes/RecipeComponent.kt
package com.example.floating.bakingapp.recipes

import dagger.Subcomponent

/**
 * Created by apple on 14/01/2018.
 */
@RecipeScope
@Subcomponent(modules = [(RecipeModule::class)])
interface RecipeComponent {
    fun inject(recipeActivity: RecipeActivity)
}
